frontend/src/helpers/lanSharing.ts

```diff
@@ -31,10 +31,13 @@ export function maskIPClass(privateIP: ipAddress, type: ipClass) {
   switch (type) {
     case 'A':
       parts[1] = '0'
+      break
     case 'B':
       parts[2] = '0'
+      break
     case 'C':
       parts[3] = '0'
+      break
   }
```

Comment:
I know TypeScript doesn't like this, but adding these `break` lines will break the functionality of this function. It will have to be re-written if it's not allowing it.
```suggestion
    case 'B':
      parts[2] = '0'
    case 'C':
      parts[3] = '0'
  }
```